<launch>

	<param name="use_sim_time" value="true"/>

<<<<<<< Updated upstream
<node pkg="rosbag" type="play" name="player" output="screen" args="--clock /home/lorenzo/progetto/bag1.bag"/>
=======
<node pkg="rosbag" type="play" name="player" output="screen" args="--clock /home/davide/project1_git/src/bag1.bag"/>
>>>>>>> Stashed changes

<node pkg="sincronizzazione" type="sincronizzazione" name="sincronizzatore" output="screen"/>

<node pkg="baseline" type="baseline" name="gen_baseline" output="screen"/>

<node pkg="generazione" type="generazione" name="generatore" output="screen"/> 

<node pkg="integrazione" type="integratore" name="integratore" output="screen"/>
	<param name="starting_x" value="0" />
	<param name="starting_y" value="0" />
	<param name="starting_omega" value="0" />


</launch>


